cumulus_library/study_parser.py

```diff
@@ -503,8 +515,37 @@ def run_single_table_builder(
         if not name.endswith(".py"):
             name = f"{name}.py"
         self._load_and_execute_builder(
-            name, cursor, schema, verbose, drop_table=True, parser=parser
+            name, cursor, schema, verbose=verbose, drop_table=True, parser=parser
+        )
+
+    def run_generate_sql(
+        self,
+        cursor: databases.DatabaseCursor,
+        schema: str,
+        parser: databases.DatabaseParser = None,
+        **kwargs,
+    ) -> None:
+        """Loads modules from a manifest and executes code via BaseTableBuilder
```

Comment:
Is that accurate? Feels like it converts jinja templates to sql more than run code.

Reply:
Reworded, lemme know what you think.